hnn_core/tests/test_network.py

```diff
@@ -17,6 +17,46 @@
 params_fname = op.join(hnn_core_root, 'param', 'default.json')
 
 
+@pytest.fixture(scope="class")
+def base_network():
```

Comment:
To @jasmainak point in https://github.com/jonescompneurolab/hnn-core/pull/745#discussion_r1555258472. It would be great of this fixture could be standardized or replaced using previously written fixtures and functions. This fixture was copied as-is and is used both in the `pick_connection` tests as well as the original `test_network_connectivity`, which still tests several other functions. I just didn't want to break any existing tests if they rely on this structure.